### ISSUE TITLE
README!!!!!

### ISSUE THREAD

**Issue by lermanori** [lermanori/media-connection#9]
this is super important!!

**Comment by reachrahul** [comment]
@lermanori can I work on this?

**Comment by lermanori** [comment]
Yes of course. This is an quasar app i did for a school project if you need
any help with something help me.
Its a little bit messy right  now.
Link:
https://media-connection.herokuapp.com
